src/qt/walletmodeltransaction.h

```diff
@@ -29,6 +29,7 @@ class WalletModelTransaction
     CTransactionRef& getWtx();
     void setWtx(const CTransactionRef&);
 
+    //! WARNING: Currently only safe for self-generated transactions due to bypassing sigop logic of GetVirtualTransactionSize
```

Comment:
seems a bit scary with no actionable suggestion?

Reply:
I don't see any obvious actionable solution (https://github.com/bitcoin-core/gui/issues/750). But this is currently only called where it is safe.